router/router.go

```diff
@@ -610,25 +610,22 @@ func (worker *workerT) handleWorkerDestinationJobs(ctx context.Context) {
 							respBodyArr = append(respBodyArr, respBodyTemp)
 						} else {
 							// stat start
-							pkgLogger.Debugf(`responseTransform status :%v, %s`, worker.rt.responseTransform, worker.rt.destName)
+							pkgLogger.Debugf(`responseTransform status :%v, %s`, worker.rt.transformerProxy, worker.rt.destName)
 							sendCtx, cancel := context.WithTimeout(ctx, worker.rt.netClientTimeout)
 							defer cancel()
-							rdl_time := time.Now()
-							resp := worker.rt.netHandle.SendPost(sendCtx, val)
-							respStatusCode, respBodyTemp, respContentType = resp.StatusCode, string(resp.ResponseBody), resp.ResponseContentType
-							// stat end
-							worker.routerDeliveryLatencyStat.SendTiming(time.Since(rdl_time))
-							//response transform start
-							if worker.rt.responseTransform {
-								dResponse := integrations.DeliveryResponseT{
-									Status: int64(respStatusCode),
-									Body:   respBodyTemp,
-								}
+							//transformer proxy start
+							if worker.rt.transformerProxy {
 								rtl_time := time.Now()
-								respStatusCode, respBodyTemp = worker.rt.transformer.ResponseTransform(ctx, dResponse, worker.rt.destName)
-								worker.routerResponseTransformStat.SendTiming(time.Since(rtl_time))
+								respStatusCode, respBodyTemp = worker.rt.transformer.ProxyRequest(ctx, val, worker.rt.destName)
+								worker.routerProxyStat.SendTiming(time.Since(rtl_time))
+							} else {
+								rdl_time := time.Now()
+								resp := worker.rt.netHandle.SendPost(sendCtx, val)
+								respStatusCode, respBodyTemp, respContentType = resp.StatusCode, string(resp.ResponseBody), resp.ResponseContentType
+								// stat end
+								worker.routerDeliveryLatencyStat.SendTiming(time.Since(rdl_time))
```

Comment:
[minor] `.Since` can be used as shortcut 
```suggestion
								worker.routerDeliveryLatencyStat.Since(rdl_time)
```